pkg/policy/rule.go

```diff
@@ -209,6 +211,27 @@ func mergeL3(ctx *SearchContext, dir string, ipRules []api.CIDR, resMap *L3Polic
 	return found
 }
 
+func computeResultantCIDRSet(cidrs []api.CIDRRule) []*net.IPNet {
+	var allResultantAllowedCIDRs []*net.IPNet
+	for _, s := range cidrs {
+		_, allowNet, err := net.ParseCIDR(string(s.Cidr))
+		if err != nil {
+			// TODO what to do with error? it should already be valdiated at this point, can we ignore?
+		}
+		var removeSubnets []*net.IPNet
+		for _, t := range s.ExceptCIDR {
+			_, removeSubnet, _ := net.ParseCIDR(string(t))
+			removeSubnets = append(removeSubnets, removeSubnet)
+		}
+		resultantAllowedCIDRs, err := ip.RemoveCIDRs([]*net.IPNet{allowNet}, removeSubnets)
+		if err != nil {
+			//TODO - error handling?
```

Comment:
All of the error conditions can be avoided by moving the code to `Validate()`. Error conditions during policy eval should be avoided at all costs as we have no choice but to lock down the entire endpoint.

Reply:
Cool, that's what I figured - wanted to make sure of that as part of my initial RFC. Thanks!

Reply:
Fixed and added comments explaining why we are ignoring errors.